kroxylicious-app/src/main/resources/log4j2.yaml

```diff
@@ -13,7 +13,7 @@ Configuration:
       name: STDOUT
       target: SYSTEM_OUT
       PatternLayout:
-        pattern: "%d{yyyy-MM-dd HH:mm:ss} <%t> %-5p %c - %m%n"
+        pattern: "%d{yyyy-MM-dd HH:mm:ss} %-5p <%t> %c{2.} sid:%X{io.kroxylicious.sessionId} - %m%n"
```

Comment:
I think you want to remove this now you've gone away from the MDC solution.

Reply:
I'd like to keep the `%-5p` and `%c{2.}` part of the change, but yes the `sid` needs to be dropped.